app/controllers/users_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      session[:user_id] = @user.id
+      redirect_to user_path(@user)
+    else
+      render :new
+    end
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :image, :password, :password_confirmation)
```

Comment:
Do we need password_confirmation in the strong params? I think this is used under the Bcrypt hood. @andrewdwooten @maxglassie